libs/cardano-data/src/Data/OMap/Strict.hs

```diff
@@ -92,7 +93,7 @@ data OMap k v = OMap
   { omSSeq :: !(SSeq.StrictSeq k)
   , omMap :: !(Map.Map k v)
   }
-  deriving (Generic, Eq)
+  deriving (Generic, Eq, Functor)
```

Comment:
1. This is not safe, because `fmap` would allow to change the key that is stored in the `v`, thus violating internal invariant where `k == v ^. okeyL`. Here is a safe implementation

```suggestion
  deriving (Generic, Eq)
  
instance Functor (OMap k) where
  fmap f = fromFoldable . fmap f . toStrictSeq
```

Reply:
It's a bit unfortunate that we named it `OMap`, it should've been called something like `OIndexMap`